package.json

```diff
@@ -2,18 +2,20 @@
   "name": "yscope-log-viewer",
   "version": "0.1.0",
   "description": "A tool tailored for viewing logs, including log formats designed by YScope.",
-  "main": "src/index.tsx",
+  "main": "src/main.tsx",
   "scripts": {
-    "analyze:size": "webpack --config webpack.prod.js --profile --json=bundle-stats.json && webpack-bundle-analyzer bundle-stats.json dist/",
-    "build": "webpack --config webpack.prod.js",
+    "analyze:size": "npx vite-bundle-visualizer",
```

Comment:
Do we want to add vite-bundle-visualizer as a dev dependency?

Reply:
I thought about and believe we don't have to, although we previously did for the webpack equivalent. That I didn't add it into the `devDependencies` list is because not every developer needs to do bundle analysis. The only benefit I can think of about adding this to the list is about dep version locking, which may help if the tool ever changes its rule of counting the bytes / presenting the results - why may cause inconsistencies with stats in any GitHub issue reported.

If you think it's important to lock the version of this analysis tool, i'll add it into the `devDependencies` list.

Reply:
I don't mind if we don't lock the version, but if it's not a dev dependency, perhaps we should document that `npm run analyze:size` will prompt to install it? Otherwise it might surprise developers.

Reply:
Updated as
> To generate a bundle analysis report, run:
> 
> ```shell
> # You may be prompted to install `vite-bundle-visualizer`. Accept the prompt to install it.
> npm run analyze:size
> ```